### PR TITLE
signup form and reusable elements

### PR DESCRIPTION
**Pull request by frusciante189**

### CODE REVIEW

**Comment by ma9o** [comment]
Looks good! Just one thing: I would keep the components and their stories directly inside the `/components` directory, without making a subfolder for every component.

**Comment by ma9o** [comment]
One last thing before we merge: could you change the way you define the components from anonymous arrow functions to normal functions? 

```javascript
// instead of 
const Component = () => { return ... }
// use this
function Component() { return ... }
```

This makes it much easier to debug since named functions show up in all stack traces. Also in the rest of the code I generally prefer using functions so we should be consistent. Is there a specific reason why you use arrow functions?

**Comment by frusciante189** [comment]
> One last thing before we merge: could you change the way you define the components from anonymous arrow functions to normal functions?
> 
> ```js
> // instead of 
> const Component = () => { return ... }
> // use this
> function Component() { return ... }
> ```
> 
> This makes it much easier to debug since named functions show up in all stack traces. Also in the rest of the code I generally prefer using functions so we should be consistent. Is there a specific reason why you use arrow functions?

No specific reason, it's just a habit and became a shortcut for me. I changed the components and will be using functions from now on

File: apps/spa-frontend/src/app/components/AuthenticationForm.stories.tsx [hunk 1]
```diff
@@ -21,9 +21,11 @@ export default {
 
 export const SignUp: StoryObj<AuthenticationFormProps> = {
   args: {
-    handleSubmit: (email, password) => {
+    handleSubmit: function (email, password) {
```
Comment (ma9o) [inline apps/spa-frontend/src/app/components/AuthenticationForm.stories.tsx:24]: I mean arrow functions are fine in cases like this. I meant rather when defining components.

**Pull request merged**